prov/verbs/src/verbs_domain.c

```diff
@@ -316,17 +316,12 @@ fi_ibv_domain(struct fid_fabric *fabric, struct fi_info *info,
 {
 	struct fi_ibv_domain *_domain;
 	struct fi_ibv_fabric *fab;
-	struct fi_info *fi;
 	int param = 0, ret;
 
-	fi = fi_ibv_get_verbs_info(info->domain_attr->name);
-	if (!fi)
-		return -FI_EINVAL;
-
 	fab = container_of(fabric, struct fi_ibv_fabric,
 			   util_fabric.fabric_fid);
 	ret = ofi_check_domain_attr(&fi_ibv_prov, fabric->api_version,
-				    fi->domain_attr, info->domain_attr);
+				    fab->info->domain_attr, info->domain_attr);
```

Comment:
here is an issue: in original implementation corresponding fi is selected (based on hints type), now first available info is selected & assumed that it holds for valid attributes. this statement is incorrect because verbs provider implements 2 types of EP's: msg & rdm